src/AbstractTypes.jl

```diff
@@ -31,15 +35,15 @@ abstract type FieldElem <: RingElem end
 
 # parameterized domains
 
-abstract type Module{T} <: Group end
+abstract type Module{T} <: AdditiveGroup end
```

Comment:
@fieker I meant this when I wrote "But the Module/ModuleElem types as well as code in Hecke violate this,"  (which BTW is not meant as an accusation, I am just bad at choosing words...)